sources/metadata/everest/0.11.0-rc1.yaml

```diff
@@ -1 +1,3 @@
 version: 0.11.0-rc1
+supported:
+  cli: 0.11.0-rc1
```

Comment:
Should this be exactly `0.11.0-rc1` or anything greater `0.11.0`? I think this actually depends on [this discussion](https://github.com/percona/everest/pull/382#discussion_r1631036481).

Reply:
Yeah, makes sense to do `>= 0.11.0`. I didn't consider that users can use a newer version of the CLI to upgrade/install an older version